test/helpers

```diff
@@ -2,24 +2,82 @@
 
 ### Helpers functions
 
-test::helpers::get_php_version() {
-    local old_home
+test::helpers::test_deploy() {
+    # Deploy a PHP app in a Scalingo runtime and test that everything is as
+    # expected. The different fixtures allow us to test different setup and
+    # conditions.
 
-    old_home="${HOME}"
+    local fixture_name="${1}"
+    local detect_expectation="${2}"
+    local version="${3}"
```

Comment:
issue: by reading these lines, it's not really clear what the 3 variables should contain. Could you make it explicit in the comment? Especially because of the `version` variable that does not seem to contain a complete version which seems really odd